contracts/JBETHERC20ProjectPayer.sol

```diff
@@ -218,15 +217,19 @@ contract JBETHERC20ProjectPayer is IJBProjectPayer, Ownable {
     // If this terminal's token is ETH, send it in msg.value.
     uint256 _payableValue = _token == JBTokens.ETH ? _amount : 0;
 
-    // Send funds to the terminal.
-    _terminal.pay{value: _payableValue}(
-      _amount, // ignored if the token is JBTokens.ETH.
-      _projectId,
-      _beneficiary,
-      _minReturnedTokens,
-      _preferClaimedTokens,
-      _memo,
-      _metadata
-    );
+    // Pay if there's a beneficiary to receive tokens.
+    if (_beneficiary != address(0))
+      // Send funds to the terminal.
+      _terminal.pay{value: _payableValue}(
+        _amount, // ignored if the token is JBTokens.ETH.
+        _projectId,
+        _beneficiary,
+        _minReturnedTokens,
+        _preferClaimedTokens,
+        _memo,
```

Comment:
calldata (same as in terminal.pay),

Reply:
making it calldata in _pay makes it weird calling from the `receive()` fn 
<img width="833" alt="image" src="https://user-images.githubusercontent.com/77952627/160878367-6b4cf0cb-aa29-449b-a435-f278da4e6da2.png">

Reply:
changed them in `pay` tho ayyayaya